matRadGUI.m

```diff
@@ -941,7 +947,16 @@ function UpdatePlot(handles)
     figure(figHandles(idxHandle));
     
     % plot physical dose
-    mPhysDose = Result.('physicalDose'); 
+    Content = get(handles.popupDisplayOption,'String');
+    SelectedCube = Content{get(handles.popupDisplayOption,'Value')};
+    if sum(strcmp(SelectedCube,{'physicalDose','effect','RBExDose','alpha','beta','RBE'})) > 0
+         Suffix = '';
+    else
+        Suffix = strsplit(SelectedCube,'_');
```

Comment:
Didnt we run into problems with the strsplit function with downwards compatibility?

Could be replaced by find(SelectedCube == '_')?!